osd/rosa_cert_signed_by_unknown_authority.json

```diff
@@ -0,0 +1,7 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "summary": "Invalid TLS certificate",
+    "description": "Your cluster uses an invalid TLS certificate that is signed by unknown authority. Invalid TLS certificates result in failing requests to applications deployed to the cluster (for example the OpenShift Console) and will impact Red Hat's ability to monitor and support your cluster. To use a custom domain please follow the documentation on setting up custom application domains at https://access.redhat.com/articles/5599621.",
```

Comment:
Please add "an" on first line between "signed by" and "unknown authority", then I'm +1